ui_testing/pages/base_pages.py

```diff
@@ -645,3 +645,12 @@ def close_connection_with_database(self, db):
     def get_current_year(self):
         current_year = datetime.datetime.now()
         return str(current_year.year)
+
+    def Clicking_on_checkboxes_in_configure_table(self,field_one,field_two,child=False):
```

Comment:
rename it to ` deselect_specific_active_table_configuration` and make replace fields field one and two by : fields = []